### DISCUSSION TITLE
如何在非证明环境如命题、例子中使用结束符？

### DISCUSSION THREAD

**Discussion by grwei** [sjtug/SJTUThesis#653] [提问] [answered]
LATEX小白提问：如何在非证明环境如命题、例子中使用结束符？感谢。

**Comment by skyzh** [comment, marked as answer]
参考 https://tex.stackexchange.com/questions/327080/newtheorem-with-qed-symbol ，可以通过 newtheorm 包新建一个环境，带上 QED 符号。

**Reply by AlexaraWu** [reply]
要不要考虑把模板里预定义的定理环境移到 `setup.tex` 里暴露给用户？感觉这个定制化需求还挺多的。

**Reply by skyzh** [reply]
感觉上可以.jpg
